service/controller/controllercontext/status.go

```diff
@@ -1,9 +1,20 @@
 package controllercontext
 
 type ContextStatus struct {
-	Worker ContextStatusWorker
+	// Versions is a map of key value pairs where the map key is a version label
+	// of a given operator. The map value is the version of the corresponding
+	// operator. See also the operatorversions resource.
+	//
+	//     aws-operator.giantswarm.io/version: 6.5.0
+	//
+	Versions map[string]string
+	// Worker is a map of key value pairs where the key is the machine deployment
+	// ID. The map value is a structure holding node information for the
+	// corresponding machine deployment.
+	Worker map[string]ContextStatusWorker
```

Comment:
Are you sure this needs to change?

Reply:
This is just moved from `clusterapi/v22` version, now with your PR to delete the obsolete code separated this diff should actually be clearer :tada: :smile: